pkgs/by-name/fl/flashprog/package.nix

```diff
@@ -14,12 +14,12 @@
 
 stdenv.mkDerivation {
   pname = "flashprog";
-  version = "1.0.1";
+  version = "1.1";
 
   src = fetchgit {
     url = "https://review.sourcearcade.org/flashprog";
-    rev = "2ca11f9a4101ea230081d448ab2b570425b7f0bd";
-    hash = "sha256-pm9g9iOJAKnzzY9couzt8RmqZFbIpKcO++zsUJ9o49U=";
+    rev = "20390c997bf60e5522c35dd041eb19b3c35ac0db";
```

Comment:
I would suggest to use the v1.1 tag as a rev here instead as it is more transparent to the user and easier to see that the v1.1 commit is referenced. Besides LGTM.

Reply:
Done.